src/hubbleds/assets/custom.css

```diff
@@ -202,6 +202,13 @@ footer .v-card__text {
     padding: 6px;
 }
 
+/* Hide the footer on a short screen */
+@media (max-height: 600px) {
+  footer.v-footer {
+    display: none;
+  }
+}
+
```

Comment:
```suggestion
/* also hide on narrow screens */
@media (max-width: 600px) {
  footer.v-footer {
    display: none;
  }
}
/* at 950px the lines wrap to be 3+ lines*/
@media (max-width: 950px) {
  .cosmicds-footer .v-card__text {
      line-height: 1;
  }
}
```

Reply:
We also need to hide in when the screen gets too narrow. To make it fit a bit better, I have adjust the `line-height` to 1 on medium-narrow screens